File: Domain/Doctor.cs
using System;

namespace Domain
{
    public class Doctor
    {
        public Guid id { get; set; }
        public string name { get; set; }
        public string surname { get; set; } 
        public DateTime dateofbirth { get; set; }
        public char gender { get; set; }
        public string street_address { get; set; }
        public string city { get; set; }
        public string country { get; set; }
        public string postal_code { get; set; }
        public string phone { get; set; }
        public string designation { get; set; }
  

        
    }
}

File: client-app/src/app/models/doctor.ts
export interface Doctor {
    id: string;
    name: string;
    surname: string;
    dateofbirth: Date | null;
    gender: string;
    street_address: string;
    city: string;
    country: string;
    postal_code: string;
    phone: string;
    designation : string;
}